spec/event_parser_spec.rb

```diff
@@ -5,10 +5,10 @@
 
   it "parses an event with all fields" do
     lines = [
-      "event: abc\r\n",
-      "data: def\r\n",
-      "id: 1\r\n",
-      "\r\n"
```

Comment:
BufferedLineReader now removes the line terminators before the lines get fed into EventParser.